src/api/db/schema.rb

```diff
@@ -771,6 +771,7 @@
     t.boolean "rss", default: false
     t.boolean "web", default: false
     t.datetime "last_seen_at", precision: nil
+    t.string "type"
```

Comment:
This file is not up to date. This should include the index and a more recent date, right?

Reply:
Hm, you're right about the index... weird because it's present in the annotations... Let me take a look

Reply:
Fixed